pubspec.lock

```diff
@@ -208,6 +288,22 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "14.2.5"
+  web:
+    dependency: transitive
+    description:
+      name: web
+      sha256: cd3543bd5798f6ad290ea73d210f423502e71900302dde696f8bff84bf89a1cb
+      url: "https://pub.dev"
+    source: hosted
+    version: "1.1.0"
+  xml:
+    dependency: transitive
+    description:
+      name: xml
+      sha256: b015a8ad1c488f66851d762d3090a21c600e479dc75e68328c52774040cf9226
+      url: "https://pub.dev"
+    source: hosted
+    version: "6.5.0"
```

Comment:
Are we using these dependencies?

Reply:
Yes, These dependencies are being used in this project. When we installed the **flutter_svg** package it implicitly added these two on **pubspec.lock**. Example below...

flutter_svg 2.0.16
│   ├── http 1.2.2
│   │   ├── http_parser 4.0.2
│   │   │   ├── typed_data 1.4.0
│   │   │   │   └── collection...
│   │   │   ├── collection...
│   │   │   ├── source_span...
│   │   │   └── string_scanner...
│   │   ├── web 1.1.0
│   │   ├── async...
│   │   └── meta...
│   ├── vector_graphics 1.1.15
│   │   ├── flutter...
│   │   ├── http...
│   │   └── vector_graphics_codec...
│   ├── vector_graphics_codec 1.1.12
│   ├── vector_graphics_compiler 1.1.16
│   │   ├── args 2.6.0
│   │   ├── path_parsing 1.1.0
│   │   │   ├── meta...
│   │   │   └── vector_math...
│   │   ├── xml 6.5.0
│   │   │   ├── petitparser 6.0.2
│   │   │   │   └── meta...
│   │   │   ├── collection...
│   │   │   └── meta...
│   │   ├── meta...
│   │   ├── path...
│   │   └── vector_graphics_codec...
│   └── flutter...